src/main/java/net/mcreator/element/ModElementTypeRegistry.java

```diff
@@ -58,6 +58,10 @@
 		put(ModElementType.LOOTTABLE, new ModTypeRegistration<>('q', 26, LootTableGUI::new, LootTable.class));
 		put(ModElementType.FUNCTION, new ModTypeRegistration<>('u', 27, FunctionGUI::new, Function.class));
 		put(ModElementType.MUSICDISC, new ModTypeRegistration<>(null, 28, MusicDiscGUI::new, MusicDisc.class));
+
+		////////////////////////////////////////FEDER DID THIS!!!

```

Comment:
Please remove this. Credit will be given on the GitHub on the file, it will show your authorship. Also keep original authorship.